<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QRadioButton" name="buger_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡腿堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buger_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>巨无霸牛堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buger_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>鳕鱼堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="danta">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>蛋挞</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="guopai">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>果派</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="shutiao">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>薯条</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cola">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>可乐</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="milk">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>牛奶</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fenda">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>芬达</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>肯德基点餐系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>汉堡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>小食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>饮品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sumit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交订单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
